src/braket/circuits/noises.py

```diff
@@ -0,0 +1,252 @@
+from typing import Iterable
+
+import numpy as np
+
+import braket.ir.jaqcd as ir
+from braket.circuits import circuit
+from braket.circuits.instruction import Instruction
+from braket.circuits.noise import Noise, ProbabilityNoise
+from braket.circuits.quantum_operator_helpers import (
+    is_CPTP,
+    verify_quantum_operator_matrix_dimensions,
+)
+from braket.circuits.qubit_set import QubitSet, QubitSetInput
+
+"""
+To add a new noise:
+    1. Implement the class and extend `Noise`
+    2. Add a method with the `@circuit.subroutine(register=True)` decorator. Method name
+       will be added into the `Circuit` class. This method is the default way
+       clients add this noise to a circuit.
+    3. Register the class with the `Noise` class via `Noise.register_noise()`.
+"""
+
+
+class Bit_Flip(ProbabilityNoise):
```

Comment:
Python class naming convention is "BitFlip", same for the other classes

Reply:
Fixed the class names. How about the circuit method Circuit.bit_flip() when adding this noise to a circuit? Should it be "Circuit().bit_flip()" or "Circuit().bitflip()" ?

Reply:
For method names, it should conform to the Python standard, so bit_flip()